middleware/index.js

```diff
@@ -1,41 +1,17 @@
-const moment = require('moment');
-
 const Models = require('../models');
 const Performance = new Models.Performance();
 const { logger } = require('../utils');
 
-function currentPerformanceWithinTimeFrame(performance) {
-  const now = new Date().toJSON();
-
-  return moment(performance.closeAt).add({ days: 1 }).isBefore(moment(now));
-}
-
-const currentPerformance = (req, res, next) => {
-  Performance.findCurrent()
-  .then((performance) => {
-    const now = moment(), { openat, closeat } = performance;
-
-    if (moment(openat).isBefore(now) && now.isBefore(moment(closeat))) {
-      req.session.currentPerformance = performance;
-    }
-    next();
-  })
-  .catch((err) => {
-    logger.errorLog('Middleware.currentPerformance', err);
-    next();
-  });
-};
-
 /* eslint-disable consistent-return */
```

Comment:
Good call. I was returning it so eslint would stop yelling at me for inconsistent return. I don't think the return value of the middleware callback affects anything, but a quick read of the docs makes me think that convention is to just call `next()`